screen-capture/META.yml

```diff
@@ -0,0 +1,3 @@
+spec: https://w3c.github.io/mediacapture-screen-share/
+suggested_reviewers:
+  - martinthomson
```

Comment:
@martinthomson are you OK with getting reviews in this directory. @alvestrand I suspect it might make sense for you to keep an eye on this, can I add you too?

Reply:
@suhasHere, I've sent an invite to the web-platform-tests org. You'll need to accept that or it won't be possible for wpt-pr-bot to assign you as a reviewer to new PRs, so please accept the invite at the top of https://github.com/web-platform-tests.